burr/integrations/opentelemetry.py

```diff
@@ -511,6 +515,91 @@ def initialize_tracer():
     trace.get_tracer_provider().add_span_processor(BurrTrackingSpanProcessor())
 
 
+INSTRUMENTS_SPECS = {
```

Comment:
Let's add this to any examples that do OpenAIInstrumentor().instrument(). Or create a new example -- litellm would be good